app/com/linkedin/drelephant/analysis/SeverityThresholds.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.linkedin.drelephant.analysis
+
+import com.linkedin.drelephant.util.Utils
+
+
+/**
+  * A convenience case class for containing severity thresholds and calculating severity.
+  */
+case class SeverityThresholds(low: Number, moderate: Number, severe: Number, critical: Number, ascending: Boolean) {
+  if (ascending) {
+    require(low.doubleValue <= moderate.doubleValue)
```

Comment:
The old code used equal numbers for severe and critical for some thresholds, e.g. https://github.com/linkedin/dr-elephant/blob/dad905cdafa6aa7665059917a455cb601155fbd1/app/com/linkedin/drelephant/spark/heuristics/JobRuntimeHeuristic.java#L47. Also, https://github.com/linkedin/dr-elephant/blob/dad905cdafa6aa7665059917a455cb601155fbd1/app/com/linkedin/drelephant/analysis/Severity.java#L150, for example, will check for equal values.